core/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -51,8 +48,20 @@ object RedundantBraces extends Rewrite {
 
   override def rewrite(code: Tree, ctx: RewriteCtx): Seq[Patch] = {
     import ctx.tokenTraverser._
+    import ctx.style.rewrite.{redundantBraces => settings}
     val builder = Seq.newBuilder[Patch]
     code.collect {
+      case t: Term.Interpolate if settings.stringInterpolation =>
+        t.parts.tail.zip(t.args).collect {
+          case (Lit(value), arg @ Term.Name(name)) =>
+            val openBrace = prevToken(arg.tokens.head)
+            val closeBrace = nextToken(arg.tokens.head)
+            (openBrace, closeBrace, value) match {
+              case (LeftBrace(), RightBrace(), "") =>
```

Comment:
Whoops, testing without braces is not so interesting I agree. What if those examples are expected outputs and the inputs have variables wrapped in braces? I still don't understand why it should only rewrite when value is ""